test/fixtures/api_keys.yml

```diff
@@ -0,0 +1,4 @@
+FakeRadius:
+  id: 1
+  bearer_name: "FakeRadius"
+  api_key: <%= Rails.application.credentials.encrypted_key! %>
```

Comment:
It's a test key, I think it's fair to have it in clear text here (with the original key as a comment above, to reuse in tests if needed)(until we create an abstraction over this of course ^^)